### PR TITLE
The remaining files

### PR DESCRIPTION
**Pull request by olakunle-exxdee**
these are the remaining files for the projects

### CODE REVIEW

**Review by bahdcoder — CHANGES_REQUESTED** [review]
Overall, amazing work on this project. You've done a ton of great work here. 

About your responsiveness, can you please check how your site looks on extra large screens like TV ?  Tell me what you think about that.

Looking forward to your changes. 

Also, please try spacing out your HTML a little more. The more readable, the better.

File: index.html [hunk 1]
```diff
@@ -1,11 +1,343 @@
...
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Classic Resume</title>
-</head>
-<body>
-    
-</body>
+  <head>
```
[the remark this answers is not in the document]
Comment (olakunle-exxdee) [reply index.html:3]: thanks

File: index.html [hunk 2]
```diff
@@ -1,11 +1,343 @@
...
+      content="https://ahrefs.com/blog/wp-content/uploads/2019/12/fb-how-to-become-an-seo-expert.png"
+    />
+    <meta property="twitter:image:width" content="1200" />
+    <meta property="twitter:image:height" content="630" />
+
+    <title>Professional Resume Website by Olakunle Exxdee</title>
+    <link rel="stylesheet" href="./normalize.css" />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
```
[the remark this answers is not in the document]
Comment (olakunle-exxdee) [reply index.html:58]: i will add the changes

File: style.css [hunk 3]
```diff
@@ -0,0 +1,372 @@
+@import url("https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;500;700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=PT+Sans+Narrow&display=swap");
+
+:root {
```
[the remark this answers is not in the document]
Comment (olakunle-exxdee) [reply style.css:4]: thanks

File: normalize.css [hunk 4]
```diff
@@ -0,0 +1,349 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```
[the remark this answers is not in the document]
Comment (olakunle-exxdee) [reply normalize.css:1]: to be honest, I heard about normalize CSS through you, most times I just reset all the CSS  from  the root  then start my projects, I don't know of any alternatives now but I will research on it

File: index.html [hunk 5]
```diff
@@ -1,11 +1,362 @@
...
+            <p>Designer / Developer</p>
+            <h1>Matt <br />McDonald</h1>
+            <button class="btn">Contact Me</button>
+          </div>
+        </header>
+      </div>
+    </section>
+    <!-- ------------------------------------------------ HEADER ---------------------------------------------- -->
+    <!-- ------------------------------------------------ BIO ---------------------------------------------- -->
+    <div class="bio-container">
```
Comment (bahdcoder) [inline index.html:126]: Your HTML reads much better right now. Well done. `bio-container -> wrapper -> bio -> bio-content -> heading`

**Pull request merged**